.appveyor.yml

```diff
@@ -1,5 +1,7 @@
 # AppVeyor.com is a Continuous Integration service to build and run tests under
 # Windows
+image:
+- Visual Studio 2017
```

Comment:
I'm not sure conda is running Visual Studio 2017. Can you confirm that we would be able to ship this package with conda?